utils/browser_helper.py

```diff
@@ -0,0 +1,47 @@
+from loguru import logger
+from selene import be
+from selene.core.exceptions import TimeoutException
+from utils.logging import Logger
+
+
+class ElementInteractions:
+    Logger.set_logger()
+
+    def __init__(self, browser):
+        self.browser = browser
+
+    def find_element(self, selector):
+        try:
+            element = self.browser.element(selector).should(be.visible)
```

Comment:
generally should is used as an assertion, why  is it used in browser_helper?

Reply:
Wouldn't it be better to first check if an element is clickable before clicking it?